pkg/kube/inject/inject.go

```diff
@@ -302,9 +302,17 @@ func injectRequired(ignored []string, config *Config, podSpec *corev1.PodSpec, m
 // ProxyImage constructs image url in a backwards compatible way.
 // values based name => {{ .Values.global.hub }}/{{ .Values.global.proxy.image }}:{{ .Values.global.tag }}
 func ProxyImage(values *opconfig.Values, image *proxyConfig.ProxyImage, annotations map[string]string) string {
-	imageName := "proxyv2"
```

Comment:
I think this line does not need to be changed.

Reply:
I moved it down to be closer to where it was operated on (like how `tag` precedes the calculation of the value). Can revert that change.